packages/ragbits-document-search/src/ragbits/document_search/documents/sources/web.py

```diff
@@ -70,12 +70,12 @@ async def list_sources(cls, uri: str) -> Sequence["WebSource"]:
         List the file under the given URI.
 
         Arguments:
-            uri: The URI to the file.
+            uri: The URI to the file. Needs to include the protocol.
```

Comment:
This should be `url`. Unlike `from_uri`, in `list_sources` the types of arguments are source-specific (different for each source), and in case of `WebSource`, a proper URL is needed.